dev/build_release.py

```diff
@@ -529,6 +511,14 @@ def init_argument_parser(cls, parser):
         '--bintray_repo', default='',
         help='Publish to this bintray repo.\n'
              'This requires BINTRAY_USER and BINTRAY_KEY are set.')
+      parser.add_argument(
```

Comment:
First part makes sense. I'm confused at the second.

The jar repo is read/write as long as you have a valid key with permission to that repo or it is public. I agree that everyone shouldn't be able to publish to the official repo, but I don't understand why users would want to read from the official repo in this case. This is for building from source, not copying upstream, prebuilt artifacts. As you put it, this is for building artifacts and writing to `bintray_repo` and `jar_repo`. Consuming artifacts from those repos is a separate concern.